### DISCUSSION TITLE
What is the unit for page dimensions?

### DISCUSSION THREAD

**Discussion by faltenberg** [Hopding/pdf-lib#1526] [Q&A]
In `src/api/sizes.ts` the A4 page format is defined with the dimension of 595.28 x 841.89. But what are the units? It is definitely not millimeters, nor is that in inches. When using `addPage()` we can specify the page dimensions as well. But in which units is that expressed?

**Comment by satyajitnayk** [comment]
@faltenberg `pdf-lib` library represent dimensions of page sizes in **points**. 
In the PDF coordinate system, **72 points** equal **1 inch**. 
So for **'4A0'**, the dimensions **[4767.87, 6740.79]** would be **4767.87 points wide and 6740.79 points high.**
